### ISSUE TITLE
Infinite loop of crashes on start

### ISSUE THREAD

**Issue by relan** [TeamNewPipe/NewPipe#5017]
### Checklist

- [x] I am using the latest version - 0.20.4 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.  Might be the same as #4998, but I didn't press the back button.
- [x] I have read the contribution guidelines. I'm reporting a crash here because it looks like a problem with the crash reporter.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1. Start NewPipe from the launcher icon.

### Actual behaviour

The splash screen pops for a fraction of a second, disappears, appears again, etc. The most frustrating thing that I could stop this only via `adb shell am force-stop org.schabi.newpipe`.

### Expected behavior

The main activity should be displayed.

### Logs

```
11-27 19:25:10.511  3658  3729 I ActivityManager: Start proc 31715:org.schabi.newpipe/u0a157 for activity {org.schabi.newpipe/org.schabi.newpipe.report.ErrorActivity}
11-27 19:25:10.564  3170  3170 D QTI PowerHAL: notify_prm_interaction
11-27 19:25:10.570 31715 31715 I MultiDex: VM with version 2.1.0 has multidex support
11-27 19:25:10.570 31715 31715 I MultiDex: Installing application
11-27 19:25:10.570 31715 31715 I MultiDex: VM has multidex support, MultiDex support library is disabled.
11-27 19:25:10.574 31715 31715 I ACRA    : ACRA is enabled for org.schabi.newpipe, initializing...
11-27 19:25:10.622 31715 31715 D NetworkSecurityConfig: No Network Security Config specified, using platform default
11-27 19:25:10.630 31715 31715 W .schabi.newpip: Accessing hidden field Ljava/util/Collections$SynchronizedCollection;->mutex:Ljava/lang/Object; (greylist-max-o, reflection, denied)
11-27 19:25:10.630 31715 31715 W .schabi.newpip: Accessing hidden method Ljava/util/Collections$SynchronizedSet;-><init>(Ljava/util/Set;Ljava/lang/Object;)V (greylist-max-o, reflection, denied)
11-27 19:25:10.630 31715 31715 W .schabi.newpip: Accessing hidden method Ljava/util/Collections$SynchronizedCollection;-><init>(Ljava/util/Collection;Ljava/lang/Object;)V (greylist-max-o, reflection, denied)
11-27 19:25:10.729  3339 31743 E ResolverController: No valid NAT64 prefix (100, <unspecified>/0)
11-27 19:25:10.778 31715 31715 D AndroidRuntime: Shutting down VM
11-27 19:25:10.778 31715 31715 E AndroidRuntime: FATAL EXCEPTION: main
11-27 19:25:10.778 31715 31715 E AndroidRuntime: Process: org.schabi.newpipe, PID: 31715
11-27 19:25:10.778 31715 31715 E AndroidRuntime: java.lang.IllegalAccessError: Method 'void org.schabi.newpipe.report.ErrorInfo.<init>(org.schabi.newpipe.report.UserAction, java.lang.String, java.lang.String, int)' is inaccessible to class 'org.schabi.newpipe.report.ErrorInfo$Creator' (declaration of 'org.schabi.newpipe.report.ErrorInfo$Creator' appears in base.apk)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at org.schabi.newpipe.report.ErrorInfo$Creator.createFromParcel(Unknown Source:39)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Parcel.readParcelable(Parcel.java:2973)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Parcel.readValue(Parcel.java:2866)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Parcel.readArrayMapInternal(Parcel.java:3244)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:292)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.BaseBundle.unparcel(BaseBundle.java:236)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Bundle.getParcelable(Bundle.java:951)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.content.Intent.getParcelableExtra(Intent.java:7931)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at org.schabi.newpipe.report.ErrorActivity.onCreate(ErrorActivity.java:209)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7825)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7814)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:107)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7356)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
11-27 19:25:10.778 31715 31715 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
11-27 19:25:10.778 31715 31715 E ACRA    : ACRA caught a IllegalAccessError for org.schabi.newpipe
11-27 19:25:10.778 31715 31715 E ACRA    : java.lang.IllegalAccessError: Method 'void org.schabi.newpipe.report.ErrorInfo.<init>(org.schabi.newpipe.report.UserAction, java.lang.String, java.lang.String, int)' is inaccessible to class 'org.schabi.newpipe.report.ErrorInfo$Creator' (declaration of 'org.schabi.newpipe.report.ErrorInfo$Creator' appears in base.apk)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at org.schabi.newpipe.report.ErrorInfo$Creator.createFromParcel(Unknown Source:39)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Parcel.readParcelable(Parcel.java:2973)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Parcel.readValue(Parcel.java:2866)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Parcel.readArrayMapInternal(Parcel.java:3244)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:292)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.BaseBundle.unparcel(BaseBundle.java:236)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Bundle.getParcelable(Bundle.java:951)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.content.Intent.getParcelableExtra(Intent.java:7931)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at org.schabi.newpipe.report.ErrorActivity.onCreate(ErrorActivity.java:209)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.Activity.performCreate(Activity.java:7825)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.Activity.performCreate(Activity.java:7814)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Handler.dispatchMessage(Handler.java:107)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.os.Looper.loop(Looper.java:214)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at android.app.ActivityThread.main(ActivityThread.java:7356)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at java.lang.reflect.Method.invoke(Native Method)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
11-27 19:25:10.778 31715 31715 E ACRA    : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
11-27 19:25:10.838 31715 31715 I Process : Sending signal. PID: 31715 SIG: 9
11-27 19:25:10.843  6503 31765 I ACRA    : Sending report /data/user/0/org.schabi.newpipe/app_ACRA-approved/2020-11-27T19:25:10.787+03:00.stacktrace
```

### Device info

 - Android version/Custom ROM version: AOSP 10.
 - Device model: LG H930DS.

**Comment by BruceMustache** [comment]
I confirm that
Android 10
Newpipe 0.24.4
I can only stop this swiping down notifications and launching settings then newpipe stops

**Comment by B0pol** [comment]
Can you stop the loop from recent apps -> swipe away NewPipe?

**Comment by mhmdanas** [comment]
I am facing this issue too, and removing NewPipe from recent apps like @B0pol suggests does indeed stop the loop.

**Comment by johanw666** [comment]
Same here on both Android 6 and 10. It appears to be happening when I add some tracks to my playlists. Copying the database from one install to another causes that to crash too, it looks like the databae gets corrupted in the process.

**Comment by TobiGr** [comment]
fixed by #4995.
We'll release a new version shortly

**Comment by johanw666** [comment]
Well, the commits for 0.20.5 have been up for a day but nothing is there on the release page yet?
